docs/self-managed/zeebe-deployment/operations/cluster-scaling.md

```diff
@@ -0,0 +1,190 @@
+---
+id: cluster-scaling
+title: "Cluster scaling"
+description: "Scale an existing cluster by adding or removing brokers."
+---
+
+:::warning
```

Comment:
Is there anything else we should be noting here? Is this an alpha feature? If this is experimental, could it cause any other issues? Will I lose any data by trying this?

Reply:
This will be in `8.4.0-alpha1`. Since this is an experimental feature, we do not recommend to use it in production. Should we add this? Is there a standard disclaimer for such features?

Reply:
I added it to the note 👍

Reply:
Thank you @christinaausley